modules/oe_content_reference_code_field/config/install/field.storage.node.oe_reference_codes.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
Update path in the oe_content_reference_code_field to install oe_reference_codes for existing sites has to be provided. Otherwise update doesn't work.

Reply:
I already mentioned this before

Reply:
This is done.